pkgs/applications/misc/oil/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, jq, peco, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "oil-buku";
+  version = "0.3.2";
+
+  src = fetchurl {
+    url = "https://github.com/AndreiUlmeyda/oil/archive/refs/tags/${version}.tar.gz";
+    sha256 = "1nwas1v0z2sjfab8ada9k9pf4jw4jn79rzmmn4n69xlfzzf4npq6";
+  };
+
+  passthru.updateScript = [ ./update.sh ];
+
+  postUnpack = ''
+    sed -i 's/$LIBDIR/$LOCALLIB/g' oil-${version}/src/oil
+  '';
```

You could set this to `$out/lib` directly, so we don't need to create an env var with `wrapProgram`. After a quick glance at the project it looks like only `LIBDIR` is used, so you do something like this, i think:
```suggestion
  postPatch = ''
    substituteInPlace src/oil --replace \
      "LIBDIR=/usr/local/lib/oil" "LIBDIR=${placeholder "out"}/lib"
  '';
```

---

This really makes sense, Imma do it.